src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <label class="form-label" for="surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="name" class="form-label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="age">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="dateOfBirth">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label for="terms">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms" id="terms">
```

Comment:
@iLebo-stack a common rule is when you have 3 and more attributes - you need to put each on a new line:
I'm not blocking this submission, but please apply it for further assignments.
```suggestion
          <input
            type="checkbox"
            name="terms"
            id="terms"
          >
```